app/Models/Entity.php

```diff
@@ -441,18 +441,15 @@ public function hasPins(): bool
         if ($this->accessAttributes() && $this->starredAttributes()->isNotEmpty()) {
             return true;
         }
-        return (bool) ($this->pinnedFiles->isNotEmpty())
-
-
-        ;
+        return (bool) ($this->pinnedFiles->isNotEmpty());
     }
 
     /**
      * @return array|string[]
      */
     public function postPositionOptions($position = null): array
     {
-        $options = [1 => __('maps/layers.placeholders.position')];
+        $options = [null => __('characters.placeholders.dont_change'), 1 => __('maps/layers.placeholders.position')];
```

Comment:
This translation isn't related to `characters`, it would be better placed in `posts.php` along with the other translations (confused why the other fields are from map layers too haha)